ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -9,22 +9,18 @@ import UIKit
 
 public extension Colors {
     struct Contact {
-        public static var title: UIColor = textPrimary
-        public static var subtitle: UIColor = textSecondary
+        public static let title: UIColor = textPrimary
+        public static let subtitle: UIColor = textSecondary
+        public static let pressedState = UIColor(red: 0.83, green: 0.83, blue: 0.83, alpha: 0.6)
```

Comment:
we should be using semantic colors here. if it isn't defined already, it should be declared in colors.swift or asset catlog

Reply:
What do you mean by semantic colours? The colours I chose aren't defined yet in `Colors.swift` as I wasn't given a specification on which colours to use. I can define them in `Colors.swift` if that is what you're reccomending.

Reply:
all fluent gray colors needs to be from fluent gray color pallet . gray10 - gray900. if this defers from that we need to talk to design.

Reply:
I can try to find a similar colour using the grey colour pallet.